artifacts_test.py

```diff
@@ -20,6 +20,8 @@
 class ArtifactsTest(ClusterTester):
     @property
     def node(self):
+        if self.db_cluster is None or not self.db_cluster.nodes:
```

Comment:
```suggestion
        if self.db_cluster is None or getattr(self.db_cluster, 'nodes') or not self.db_cluster.nodes:
```

The original issue why the job failed:
```
'NoneType' object has no attribute 'nodes'
```

Reply:
we will catch the issue in the first check since `self.db_cluster ` will be None (nodes are initialized in the constructor).